server/data/activeLduClient.js

```diff
@@ -0,0 +1,23 @@
+const db = require('./dataAccess/db')
+
+module.exports = {
+  async addLdu(lduCode) {
+    const query = {
+      text: `INSERT INTO active_local_delivery_units (ldu_code) VALUES ($1) ON CONFLICT (ldu_code) DO UPDATE SET ldu_code=active_local_delivery_units.ldu_code RETURNING ldu_code`,
+      values: [lduCode],
+    }
+    const response = await db.query(query)
+    return response
+  },
+
+  async doesLduExist(lduCode) {
+    const query = {
+      text: `SELECT ldu_code FROM active_local_delivery_units WHERE ldu_code=$1`,
+      values: [lduCode],
+    }
+
+    const response = await db.query(query)
+    const active = response && response.rows
+    return active ? 'true' : 'false'
```

Comment:
we would want to return a boolean here not string. 
Returning strings is confusing as 'false' is truthy

Reply:
I did initially use booleans for true and false but the linter gave me the following error and that's why I changed to a string
  **21:33  error  Unnecessary use of boolean literals in conditional expression** 
I could disable linter for that line?